### ISSUE TITLE
scrollToDate on year view Calendar does nothing

### ISSUE THREAD

**Issue by zero-tolerance0** [patchthecode/JTAppleCalendar#1226]
i searched issue before starting new issue:
https://github.com/patchthecode/JTAppleCalendar/issues/1055

however solution from there did not help.

**(Required) Version Number:**
macOs 10.14 Mojave
xcode 11.3
JtAppleCalendar github "patchthecode/JTAppleCalendar" "master"
using carthage

## Description
i answered:
unfortunately function
self.yearView.scrollToItem(at: IndexPath(item: itemIndex, section: 0), at: at, animated: animated)
not scrolling calendarYear to specified IndexPath

## Expected Behavior
i want to run scroll to date f-n and expect calendar will scroll to that date, but now, it always starts from very beginning date((

## Additional Context
it is called and some strange action happens (in mili seconds of time) but calendarYear still starts from the very beginning (

scrollToItem func is definitely getting called since if I put very big row for IndexPath it crashes.
